tests/test_utils/test_config.py

```diff
@@ -429,6 +429,26 @@ def test_dict():
         assert cfg.item2.a == 1
 
 
+@pytest.mark.parametrize('file', ['a.json', 'b.py', 'c.yaml', 'd.yml', None])
+def test_dump_from_dict(file):
```

Comment:
Suggest renaming `test_dump_from_dict` to `test_dump` and add a unit test that its `self.filename` is not None.